stripe/src/main/java/com/stripe/android/paymentsheet/ui/BaseSheetActivity.kt

```diff
@@ -95,6 +106,20 @@ internal abstract class BaseSheetActivity<ResultType> : AppCompatActivity() {
         bottomSheetController.hide()
     }
 
+    private fun updateToolbarButton(isStackEmpty: Boolean) {
+        if (isStackEmpty) {
+            toolbar.navigationIcon =
+                ContextCompat.getDrawable(this, R.drawable.stripe_paymentsheet_toolbar_close)
```

Comment:
I think you have new lines after the open parens, each parameter on a new line, and the end parens on a new line

Reply:
Isn't that only for when the line goes over the 100 char limit?

Reply:
Nevermind, this does go over. Android Studio auto-format doesn't break it 😕